integration-tests/bats/foreign-keys.bats

```diff
@@ -1651,3 +1651,37 @@ SQL
     dolt add .
     dolt commit -m 'this should not break'
 }
+
+@test "foreign-keys: dolt table import with null in nullable FK field should work (issue #2108)" {

```

Comment:
Don't think we should have issue #s in bats

Reply:
My bad - I assumed it would make it easier to understand the context. I'll remember this for next time.